angrmanagement/ui/views/jobs_view.py

```diff
@@ -0,0 +1,47 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+from PySide6.QtCore import QSize
+from PySide6.QtWidgets import QVBoxLayout, QWidget
+
+from angrmanagement.ui.widgets.qjobs import QJobs
+
+from .view import InstanceView
+
+if TYPE_CHECKING:
+    from angrmanagement.data.instance import Instance
+    from angrmanagement.ui.workspace import Workspace
+
+
+class JobsView(InstanceView):
+    """
+    This creates a view for the jobs view which creates a table to display all the jobs being ran or in queue
+    """
+
+    def __init__(self, workspace: Workspace, default_docking_position: str, instance: Instance) -> None:
+        super().__init__("jobs", workspace, default_docking_position, instance)
+        self.base_caption = "Jobs"
+        self.central_widget = QWidget()
+        self.workspace = workspace
+        self._init_widgets(self.workspace)
```

Comment:
Given how little is done in the constructor, I'd consider moving the logic from the method into the constructor and removing the method. More readable that way IMO